core/fake/mutation_storage.go

```diff
@@ -44,15 +44,14 @@ func (m *MutationStorage) ReadPage(_ context.Context, domainID string, revision,
 	if !ok {
 		return 0, nil, fmt.Errorf("DomainID: %v, revision %v not found", domainID, revision)
 	}
-	if got, want := int(start), len(mutationList); got >= want {
+	if int(start) > len(mutationList) {
```

Comment:
This has changed from `>=` to `>`  -- which is correct?

Reply:
It turns out that they are almost equivalent. 
```
mutationList[len(mutationlist):] = [] ~= nil
```

I think returning `[]` makes a little more sense though, so I'll keep it as is.